src/black/handle_ipynb_magics.py

```diff
@@ -276,6 +306,21 @@ def unmask_cell(src: str, replacements: list[Replacement]) -> str:
     return src
 
 
+def _get_code_start(src: str) -> str:
+    """Provides the first line where the code starts.
+
+    Iterates over lines of code until it finds the first line that doesn't
+    contain only empty spaces and comments. It removes any empty spaces at the
+    start of the line and returns it. If such line doesn't exist, it returns an
+    empty string.
+    """
+    for match in re.finditer(".+", src):
```

Comment:
```suggestion
    for match in src.splitlines():
```
Any reason not to use this?

Reply:
There might be a slight performance difference because `re.finditer` will stop splitting string once the first line with code is found while `src.splitlines()` will always split the entire string by lines. But I haven't benchmarked anything and this probably won't affect much the performance of the entire formatting process.

I'm also ok with switching to `src.splitlines()` if that is the preferred way.

Reply:
Thanks, I guess this is fine.